app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -183,5 +207,9 @@
       border-bottom-left-radius: 0;
     }
   }
+
+  .client-navigation {
```

Comment:
The client tabs were the component that was causing the over 100% problem because it had its _own_ negative margin, positive padding inside of the slab that was doing that, and it got wonky.

Reply:
check to see if this class override should be on tab-bar